x-pack/plugins/observability_solution/uptime/server/legacy_uptime/lib/alerts/duration_anomaly.ts

```diff
@@ -181,53 +178,56 @@ export const durationAnomalyAlertFactory: UptimeAlertTypeFactory<ActionGroupIds>
         );
 
         const alertId = DURATION_ANOMALY.id + index;
-        const indexedStartedAt = getAlertStartedDate(alertId) ?? startedAt.toISOString();
-        const alertUuid = getAlertUuid(alertId);
-        const relativeViewInAppUrl = getMonitorRouteFromMonitorId({
-          monitorId: alertId,
-          dateRangeEnd: 'now',
-          dateRangeStart: indexedStartedAt,
-        });
 
-        const alert = alertWithLifecycle({
+        const { start, uuid } = alertsClient?.report({
           id: alertId,
-          fields: {
+          actionGroup: DURATION_ANOMALY.id,
+          payload: {
             'monitor.id': params.monitorId,
             'url.full': summary.monitorUrl,
             'observer.geo.name': summary.observerLocation,
             'anomaly.start': summary.anomalyStartTimestamp,
-            'anomaly.bucket_span.minutes': summary.bucketSpan,
+            'anomaly.bucket_span.minutes': `${summary.bucketSpan}`,
```

Comment:
is this change from `number` to `string` going to cause any issues?

Reply:
I didn;t get any error when i was testing but I think we better ask this to @elastic/obs-ux-infra_services-team.
`uptimeRuleTypeFieldMap` forces it to be string.
```
'anomaly.bucket_span.minutes': {
    type: 'keyword',
    required: false,
  }
 ``` 
 I can convert it to `long` but not sure about their usage.
 
 @crespocarlos can you help us here?

Reply:
OK, i changed this part and just used type casting.

Reply:
I think the mapping is wrong.
`bucket_span` is definitely number. And it has been passed as number even though defined as keyword in the mapping. It just works because JS is not type safe at run time.
I'd rather leave it as it is since the rule type is deprecated.